websubhub-examples/kafka-hub/hub/Config.toml

```diff
@@ -0,0 +1,30 @@
+[kafkaHub.config]
+# Flag to check whether to enable/disable security
+SECURITY_ON = true
+
+# Server Id which is to be used in a server cluster
+SERVER_ID = "server-1"
+
+# IP and Port of the Kafka bootstrap node
+KAFKA_BOOTSTRAP_NODE = "localhost:9092"
+
+# Kafka topic which stores websub registered topics for this server
```

Comment:
Kafka topic which stores websub registered topics. This is a shared Kafka topic among all the hubs.